tests/utils/payloads.py

```
@@ -1365,3 +1365,60 @@ def response_handler(self, response: Any) -> str:
+            )
+            return f"binary_audio_{len(audio_bytes)}_bytes"
+        result = response.json()
+        assert (
+            result.get("status") != "failed"
+        ), f"Audio generation failed: {result.get('error', 'unknown')}"
+        assert (
+            "data" in result
+        ), f"Missing 'data' in response. Keys: {list(result.keys())}"
+        assert len(result["data"]) > 0, "Empty data in audio response"
+        entry = result["data"][0]
+        if "url" in entry and entry["url"]:
```

Ruff (0.15.10)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)